### PR TITLE
Add PHP 8.4 support for pgsql

### PR DESCRIPTION
**Pull request by rafaelbernard**
- **Add PHP 8.4 support for pgsql**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,6 @@
 /vendor/
 composer.lock
 .php_cs.cache
+
+# IDEs
+.idea
```
Comment (Nyholm) [inline .gitignore:6]: This is a hard no for two reasons. 
- It belongs in your global gitignore
- Jetbrains actually recommends committing this. 

So no matter the philosophy you have about this directory, you should not add it into a project.

**Review by Nyholm — CHANGES_REQUESTED** [review]
Thank you for this PR. 

Just remove your changes in .gitignore and I'll be happy to merge.

File: .gitignore (hunk 1 above)
Comment (rafaelbernard) [reply .gitignore]: Thanks for providing the details. That is all good.

**Review by Nyholm — APPROVED** [review]
Great. Thank you

**Comment by rafaelbernard** [comment]
Thanks for reviewing and merging.

**Pull request merged**